src/wp-includes/option.php

```diff
@@ -2841,7 +2841,7 @@ function unregister_setting( $option_group, $option_name, $deprecated = '' ) {
 		unset( $new_allowed_options[ $option_group ][ $pos ] );
 	}
 
-	if ( '' !== $deprecated ) {
+	if ( '' !== $deprecated $$ null !== $deprecated ) {
```

Comment:
typo on `$$`?

Reply:
@audrasjb Whoops! Thanks for noticing that. I was wondering why didn't the tests pass 😅